Updating docs for sec issues (#338)
Aiven-Open/klaw-docs: docs/setup-configuration/klaw-installation/run-source.md

   - **Username:** `superadmin`
   - **Password:** `kwsuperadmin123$$` (for Klaw releases 2.5.0 and older)
   - **Password:** `welcometoklaw` (starting from the Klaw 2.6.0 release)

>    - **Password:** based on configuration of klaw.superadmin.default.password in core/resources/application.properties (starting from the Klaw 2.9.0 release)

Reviewer: ```suggestion
   - **Password:** Set based on the `klaw.superadmin.default.password` property in `core/resources/application.properties`. Available starting in Klaw 2.9.0.
```

Reviewer 2: klaw 2.10.0 !

>    - **Password:** Set based on the `klaw.superadmin.default.password` property in `core/resources/application.properties`. Available starting in Klaw 2.9.0.

Reviewer 2: ```suggestion
   - **Password:** Set based on the `klaw.superadmin.default.password` property in `core/resources/application.properties`. Available starting in Klaw 2.10.0.
```

>    - **Password:** Set using the `klaw.superadmin.default.password` property in the `core/resources/application.properties` file. Starting with Klaw 2.10.0, users must define their own superadmin password by setting this property.

Reviewer 2: ```suggestion
   - **Password:** Set using the `klaw.superadmin.default.password` property in the `core/resources/application.properties` file. Starting with Klaw 2.10.0, users must define their own superadmin password by setting this property, on initial install of klaw.
```

As its only needed the first time ? wdyt?

>    - **Password:** Set based on the `klaw.superadmin.default.password` property in `core/resources/application.properties`. From Klaw 2.10.0 we are enforcing users to set their own superadmin password by setting the `klaw.superadmin.default.password` property in core/resources/application.properties

Reviewer: ```suggestion
   - **Password:** Set using the `klaw.superadmin.default.password` property in the `core/resources/application.properties` file. Starting with Klaw 2.10.0, users must define their own superadmin password by setting this property.
```
